### PR TITLE
🧾 Add logging for server side

### LINKED ISSUE #60: 🎆 Add event loggin for server side
# 👮‍♂️ What's needed
Add event login for server

## 📋 Subtasks
- [x] server_event_team_created
- [x] server_event_channel_created
- [x] server_event_thread_created
- [x] server_event_reply_created
- [x] server_event_user_subscribed
- [x] server_event_user_unsubscribed
- [x] server_event_user_created
- [x] server_event_user_loaded
- [x] server_event_user_logged_in
- [x] server_event_user_logged_out
- [x] server_event_private_message_sended

# 📜 Sources
No specific source

### PR DESCRIPTION
**Pull request by 6im0n**
# :sparkles: : What's changed
Add event login for server:

- [x] server_event_team_created
- [x] server_event_channel_created
- [x] server_event_thread_created
- [x] server_event_reply_created
- [x] server_event_user_subscribed
- [x] server_event_user_unsubscribed
- [x] server_event_user_created
- [x] server_event_user_loaded
- [x] server_event_user_logged_in
- [x] server_event_user_logged_out
- [x] server_event_private_message_sended

:link: Linked issues:
- #60

### CODE REVIEW

**Review by axel-eck — DISMISSED** [review]
:cry: Sad for macOS compatibility

File: server/src/routes/login.c [hunk 1]
```diff
@@ -44,8 +47,7 @@ response_t login_route(request_t *request, void *data)
         ((json_string_t *) json_object_get(body, "username"))->value, &r);
     json_object_add(response_body, (json_t *)
         json_string_create("user_uuid", uuid_to_string(c->uuid)));
-    response_body_str = json_serialize((json_t *) response_body);
-    r.body = strdup(response_body_str);
+    r.body = strdup(json_serialize((json_t *) response_body));
```
Comment (Marius-P1) [inline server/src/routes/login.c:49]: You can delete the variable response_body_str if you don't use it anymore

**Review by Marius-P1 — CHANGES_REQUESTED** [review]
Just a little change

**Review by Marius-P1 — APPROVED** [review]
:ok_hand:

**Pull request merged**